Docs/source/install/hpc/lxplus.rst

```diff
@@ -0,0 +1,141 @@
+
+.. code-block:: bash
+
+    export WORK=/afs/cern.ch/work/<a>/<account>/
+
+GCC
+^^^
+The pre-installed GNU compiler is outdated so we need a more recent compiler. Here we use the gcc 9.2.0 from the LCG project, but other options are possible.
+
+We activate it by doing
+
+.. code-block:: bash
+
+    source /cvmfs/sft.cern.ch/lcg/releases/gcc/9.2.0/x86_64-centos7/setup.sh
+
+In order to avoid using different compilers this line could be added directly into the ``$HOME/.bashrc`` file.
+
+Spack
+^^^^^
+We download and activate Spack in ``$WORK``:
+
+.. code-block:: bash
+
+    cd $WORK
+    git clone -c feature.manyFiles=true https://github.com/spack/spack.git
+    source spack/share/spack/setup-env.sh
+
+When installing packages Spack will try to set permissions in a way which is forbidden on the LXPLUS file system (AFS), but we can avoid this by modifying the ``spack/etc/spack/defaults/config.yaml``. Inside this file we need to change the line
+
+.. code-block:: bash
+
+    allow_sgid: true
+
+to
+
+.. code-block:: bash
+
+    allow_sgid: false
+
+Now we add our gcc 9.2.0 compiler to spack:
+
+.. code-block:: bash
+
+    spack compiler find /cvmfs/sft.cern.ch/lcg/releases/gcc/9.2.0-afc57/x86_64-centos7/bin/
+
+Installing the Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With the following commands we install the dependencies for WarpX:
+
+.. code-block:: bash
+
+    spack install ncurses+symlinks %gcc@9.2.0
+    spack install adios2 ^ncurses+symlinks %gcc@9.2.0
+    spack install blaspp ^ncurses+symlinks %gcc@9.2.0
+    spack install cmake ^ncurses+symlinks %gcc@9.2.0
+    spack install ccache ^ncurses+symlinks %gcc@9.2.0
+    spack install fftw ^ncurses+symlinks %gcc@9.2.0
+    spack install hdf5 ^ncurses+symlinks %gcc@9.2.0
+    spack install lapackpp ^ncurses+symlinks %gcc@9.2.0
+    spack install mpi ^ncurses+symlinks %gcc@9.2.0
+    spack install openpmd-api ^ncurses+symlinks %gcc@9.2.0
+    spack install pkgconfig %gcc@9.2.0
+
+If we are planning on running WarpX on GPUs we also need to install Cuda:
+
+.. code-block:: bash
+
+    spack install cuda ^ncurses+symlinks %gcc@9.2.0
+
+Now we load the installed modules:
+
+.. code-block:: bash
+
+    spack load cmake
+    spack load openmpi
+    spack load ccache
+
+And if we installed Cuda:
+
+.. code-block:: bash
+
+    spack load cuda
+
+Building WarpX
+^^^^^^^^^^^^^^
+
+We download WarpX in ``$WORK``:
+
+.. code-block:: bash
+
+    cd $WORK
+    git clone https://github.com/ECP-WarpX/WarpX.git
+    cd WarpX
+
+Then we build WarpX:
+
+.. code-block:: bash
+
+    cmake -S . -B build
+    cmake --build build
+
+Or if we need to compile with cuda:
+
+.. code-block:: bash
+
+    cmake -S . -B build -DWarpX_COMPUTE=CUDA -DAMReX_CUDA_ARCH='7.0;7.5'
+    cmake --build build
+
+Python Bindings
+^^^^^^^^^^^^^^^
+
+Here we assume that a Python interpreter has been set up (e.g. with Spack or a Miniconda).
```

Comment:
```suggestion
Here we assume that a Python interpreter has been set up (e.g., with Spack or a Miniconda).
```

Reply:
I would not mix in (Mini)Conda with Spack here, since Conda brings in its own compilers and stdlibs, which likely will cause very hard to debug issues down the road.
The easiest way to build the Python bindings is to add the spack packages `python` and `py-pip`, then do the steps below.

Reply:
Ok then I'll just add `python` and `py-pip` them to the environment file I think